### ISSUE TITLE
About the CoT infer

### ISSUE THREAD

**Issue by dszpr** [pengshuai-rin/MultiMath#2]
Hi! Appreciate for the great dataset!
I tried to do evaluation for Mathvisita with Multimath-7B according to the instruction. And I noticed that '-use_multimath_prompt' is not used in the default MultiMath-main\eval_mathvista\scripts\infer.sh. And the format of the output response is "The answer is XX" without step-by-step solution.  However, the innovation point of your work is step-by-step inference. So I wonder whether you use '-use_multimath_prompt' to obtain the 50.0 metric point of Mathvista in the paper.
Looking forward to your reply!

**Comment by pengshuai-rin** [comment]
As stated in our paper, the scores on MathVista were obtained using the original prompts provided by the benchmark. We did not use CoT reasoning in MathVista inference because the training dataset, MathV360K, does not include CoT-style examples. Additionally, we found that applying the MultiMath prompt on MathVista slightly lowered the results. However, when tested on text-only benchmarks, the MultiMath prompt led to higher scores.

**Comment by dszpr** [comment]
What does 'text-based benchmarks' mean? Does it mean single-modality benchmarks that only contains text without image?

**Comment by pengshuai-rin** [comment]
> What does 'text-based benchmarks' mean? Does it mean single-modality benchmarks that only contains text without image?

yes, such as GSM8K, MATH, CMATH et al.
